embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/datasource/DataSourceImpl.kt

```diff
@@ -37,7 +37,10 @@ internal abstract class DataSourceImpl<T>(
     ): Boolean {
         try {
             if (enforceLimits && !limitStrategy.shouldCapture()) {
-                logger.logWarning("Data capture limit reached.")
+                logger.logInfo(
```

Comment:
Do we want to output this multiple times or just once per session?

Reply:
IMO let's do it multiple times for now, then following feedback from customers change to once per session if it's too spammy. I'm hoping that setting the log level & increasing descriptiveness of message will be enough of a change